### ISSUE TITLE
shields.io badges

### ISSUE THREAD

**Issue by Gama11** [HaxeFoundation/haxelib#308]
It would be nice if [shields.io](http://shields.io/) badges were available for haxelib - this is the case for a lot of other package managers like npm, NuGet, gem, etc..

There's usually a few different stats available:
- current version [![npm](https://img.shields.io/npm/v/npm.svg)]()
- downloads/month [![npm](https://img.shields.io/npm/dm/localeval.svg)]()
- downloads/total [![npm](https://img.shields.io/npm/dt/express.svg)]()

Some also have daily  / weekly downloads.

**Comment by nadako** [comment]
I like this idea, considering hipster invasion in the today's internets! However, IIUC to do that, we need to implement a REST API for haxelib (and then use it in https://github.com/badges/shields/blob/master/server.js). Maybe we could somehow autogenerate REST handlers from our `SiteApi` though.

**Comment by ibilon** [comment]
It'd probably be a lot quicker/easier to make the REST API manually.

If we need only the current version, and total/per version download (that's all we have) it's only a couple of lines.

**Comment by nadako** [comment]
I agree. Though that again raises the question of API design, since it'll be public, we may look into what NPM and others provide.

**Comment by nadako** [comment]
OTOH, we could just quickly implement a `v1` routes in the website just for mentioned things and think of "proper" API later (if ever) :)

**Comment by vladimyr** [comment]
> OTOH, we could just quickly implement a `v1` routes in the website just for mentioned things and think of "proper" API later (if ever) :)

Or someone could transpile `haxe.remoting.HttpAsyncConnection` into js and use it to make generic RPC client for nodejs → https://npm.im/haxe-rpc-client

Then it could be used to fetch relevant data from haxelib registry https://github.com/badgen/badgen.net/pull/381

![image](https://user-images.githubusercontent.com/1170440/81173233-059dfa00-8fa0-11ea-9d33-69a9d471811e.png)

Yeah, I know that badgen.net isn't really shields.io but it is probably better than nothing, right? 😉
